src/main/scripts/org/reaktivity/specification/nukleus/socks/streams/forward/client.connect.fallback.to.no.authentication/server.rpt

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()}
+property newServerConnectRef ${newClientAcceptRef} # external scope
+
+property serverAccept "nukleus://target/streams/socks#source"
+
+# Build the TcpBeginExFW using hardcoded values
```

Comment:
Just use the host and port from the route instead when encoding the connect reply (like address "example.com" port 8080). We don't need full blown socks vs tcp support at this time (Later when we do want to add that we will use SocksBeginEx plus a transformer or fuser to transform the extension metadata according to the target nukleus).